src/amo/components/AddonDetail.js

```diff
@@ -1,23 +1,37 @@
 import React, { PropTypes } from 'react';
 
-import translate from 'core/i18n/translate';
-
 import AddonMeta from 'amo/components/AddonMeta';
 import InstallButton from 'disco/components/InstallButton';
 import LikeButton from 'amo/components/LikeButton';
 import ScreenShots from 'amo/components/ScreenShots';
 import SearchBox from 'amo/components/SearchBox';
+import translate from 'core/i18n/translate';
+import { nl2br, sanitizeHTML } from 'core/utils';
 
 
 import 'amo/css/AddonDetail.scss';
 
-export class AddonDetail extends React.Component {
+class AddonDetail extends React.Component {
   static propTypes = {
     i18n: PropTypes.object,
+    addon: PropTypes.shape({
+      name: PropTypes.string.isRequired,
+      authors: PropTypes.array.isRequired,
+      slug: PropTypes.string.isRequired,
+    }),
   }
 
   render() {
-    const { i18n } = this.props;
+    const { i18n, addon } = this.props;
+    const authorList = [];
+    for (const author of addon.authors) {
+      authorList.push(<a key={author.url} href={author.url}>{author.name}</a>);
```

Comment:
Ah now I see it's in an h1, in which case it can't be a ul. Ignore the previous comment.